cmd/server-mux.go

```diff
@@ -220,7 +193,7 @@ func (l *ListenerMux) Accept() (net.Conn, error) {
 	if err != nil {
 		return conn, err
 	}
-	connMux := NewConnMux(conn)
+	connMux := NewConnMux(conn, longestWord+1)
```

Comment:
The purpose of adding one was to inclue the space which is always present after the HTTP verb, but it looks like we are not using it anymore.